sktime/forecasting/compose/_column_ensemble.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, forecasters):
         # set requires-fh-in-fit depending on forecasters
         if isinstance(forecasters, BaseForecaster):
             self.clone_tags(forecasters, "requires-fh-in-fit")
+            self.clone_tags(forecasters, "scitype:y")
         else:
             forecaster_requires_fh_in_fit = (
```

Comment:
this will break if mistakenly `forecasters` is not of the expected type without informative error message. I would check validity in the init, which is the only way to prevent this that I can see.